docs/metrics/work-in-progress.md

```diff
@@ -0,0 +1,20 @@
+# Work in progress metrics
+
+## Work in progress
+
+To provide you with an high-level view of the current work in progress of your teams, Pulse groups pull requests that are currently open by the following phases:
+
+-   **Open:** the pull request is open but the team hasn't interacted or reviewed it.
```

Comment:
```suggestion
-   **Open:** the pull request is open but the team hasn't reviewed it.
```

I think the interacting is misleading since for example normal comments do not count for this state. We are only using reviews

Reply:
Yes, your suggestion makes sense and also ties in well with the description for **In review**.